mlxtend/frequent_patterns/apriori.py

```diff
@@ -105,13 +105,12 @@ def apriori(df, min_support=0.5, use_colnames=False, max_len=None, verbose=0):
     http://rasbt.github.io/mlxtend/user_guide/frequent_patterns/apriori/
 
     """
-    allowed_val = {0, 1, True, False}
-    unique_val = np.unique(df.values.ravel())
-    for val in unique_val:
-        if val not in allowed_val:
-            s = ('The allowed values for a DataFrame'
-                 ' are True, False, 0, 1. Found value %s' % (val))
-            raise ValueError(s)
+    idxs = np.where((df.values != 1) & (df.values != 0))
```

Comment:
Do you know if that captures `True` and `False` values as well by chance?

Reply:
Yeah, in python, 1 == True (==1.0)

Reply:
haha, right, seems like my basic Python skill are getting a tad rusty